advanced_functionality/batch_transform_associate_predictions_with_input/Batch Transform - breast cancer prediction with high level SDK.ipynb

```diff
@@ -0,0 +1,468 @@
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {
+    "nbpresent": {
+     "id": "f3b125ad-a2d5-464c-8cfa-bd203034eee4"
+    }
+   },
+   "outputs": [],
+   "source": [
+    "%%time\n",
+    "from time import gmtime, strftime\n",
+    "from sagemaker.amazon.amazon_estimator import get_image_uri\n",
+    "\n",
+    "\n",
+    "job_name = 'xgb-' + strftime(\"%Y-%m-%d-%H-%M-%S\", gmtime())\n",
+    "output_location = 's3://{}/{}/output/{}'.format(bucket, prefix, job_name)\n",
+    "image = get_image_uri(boto3.Session().region_name, 'xgboost')\n",
+    "\n",
+    "sm_estimator = sagemaker.estimator.Estimator(image,\n",
+    "                                             role,\n",
+    "                                             train_instance_count=1,\n",
+    "                                             train_instance_type='ml.m5.4xlarge',\n",
+    "                                             train_volume_size=50,\n",
+    "                                             input_mode='File',\n",
+    "                                             output_path=output_location,\n",
+    "                                             sagemaker_session=sess)\n",
+    "\n",
+    "sm_estimator.set_hyperparameters(objective=\"binary:logistic\",\n",
+    "                                 max_depth=5,\n",
+    "                                 eta=0.2,\n",
+    "                                 gamma=4,\n",
+    "                                 min_child_weight=6,\n",
+    "                                 subsample=0.8,\n",
+    "                                 silent=0,\n",
+    "                                 num_round=100)\n",
+    "\n",
+    "train_data = sagemaker.session.s3_input('s3://{}/{}/train'.format(bucket, prefix), distribution='FullyReplicated', \n",
+    "                                        content_type='text/csv', s3_data_type='S3Prefix')\n",
+    "validation_data = sagemaker.session.s3_input('s3://{}/{}/validation'.format(bucket, prefix), distribution='FullyReplicated', \n",
+    "                                             content_type='text/csv', s3_data_type='S3Prefix')\n",
+    "data_channels = {'train': train_data, 'validation': validation_data}\n",
+    "\n",
+    "\n",
+    "# Start training by calling the fit method in the estimator\n",
+    "sm_estimator.fit(inputs=data_channels, logs=True)"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {
+    "collapsed": true,
+    "nbpresent": {
+     "id": "397fb60a-c48b-453f-88ea-4d832b70c919"
+    }
+   },
+   "source": [
+    "---\n",
+    "\n",
+    "## Batch Transform\n",
+    "\n",
+    "In SageMaker Batch Transform, we introduced 3 new attributes - __input_filter__, __join_source__ and __output_filter__. In the below cell, we use the Python SDK to kick-off several Batch Transform jobs using different configurations of these 3 new attributes.\n",
```

Comment:
Can we link to the documentation page as well: https://docs.aws.amazon.com/sagemaker/latest/dg/batch-transform-data-processing.html